### ISSUE TITLE
JSON parse Error 822 for any request

### ISSUE THREAD

**Issue by raymount** [mailjet/mailjet-gem#146]
I have a problem, 
i am able to send emails, but i am not able to retrieve or add people to Lists. Everytime i get the following JSON-Parse Error. DO you have any idea, why i get this error?

Example:
[1] pry(main)> Mailjet::Listrecipient.all(limit: 0)
Why is it even html? When i explicit write (format: :json) i still get this error.
Maybe i just do a stupid error, but atm, i dont see what i am doing wrong.

Thanks in advance

```
JSON::ParserError: 822: unexpected token at 
 '<html>
 <head><title>404 Not Found</title></head>
 <body bgcolor="white">
 <center><h1>404 Not Found</h1></center>
 <hr><center>nginx</center>
 </body>
</html>'
```

**Comment by jeangus** [comment]
Hello @raymount, 
I have exactly the same problem.
Have you resolved it ?

Thanks a lot !

**Comment by xab3r** [comment]
Is this still an issue? Could you please specify what gem version are you using and detailed steps you went through?

**Comment by xab3r** [comment]
Fixed in v1.6.0
